pom.xml

```diff
@@ -582,6 +582,12 @@
         <scope>compile</scope>
         <optional>true</optional>
       </dependency>
+      <!-- OS Detection for loading the native libraries -->
+      <dependency>
+        <groupId>kr.motd.maven</groupId>
+        <artifactId>os-maven-plugin</artifactId>
+        <version>1.2.3.Final</version>
+      </dependency>
```

Comment:
So sorry: I thought I responded to this, but I guess I figured my response here https://github.com/netty/netty/pull/3502/files#r26598373 covered both questions.

The dependency is very small, and by pulling it in we ensure that both sides are using the same logic to generate the artifact names. I'm not aware of any advantage we gain by copying vs. simply adding this as a dep.